data/hai/hai reveal 4 middle.txt

```diff
@@ -0,0 +1,462 @@
+		add fleet "Smuggler (Hai Traffickers)" 1000
+	
+
+mission "Hai Rescue [A04] Tarazed Analysis"
+	landing
+	name "Missing Researchers"
+	description `Find and board a ship that has microtransceivers. Your display will identify such ships as government "Smuggler." Try the uninhabited systems and pirate systems near Ultima Thule. Head to <destination> afterwards.`
+	source Wayfarer
+	destination Earth
+	clearance `Curious about your story, the authorities let you land.`
+	to offer
+		has "ZZ Hai Rescue [A03] Microtransceiver Identification: done"
+	to complete
+		has "Hai Rescue [A05] Boarded Smuggler: offered"
+	on offer
+		event "devil-run smugglers visible"
+		log "People" "Goraz Zoltof" `Goraz is a communications researcher from Tarazed Shipyards. His wife went missing some time ago and he hopes his research will help you find her.`
+		log "Factions" "Smugglers" `Some pirate or merchant ships (identified by the computer as "Smuggler") carry slaves with special microtransceivers. They are probably carrying human or Hai researchers.`
+		log `Goraz Zoltof used the galactic network of television transmitters to scan for evidence of smuggling ships carrying kidnapping victims with microtransceivers. He detected hints of transmissions in the uninhabited space between the Far North and the Syndicate, and also near pirate systems.`
+		conversation
+			`You read Teeneep's contact information for Tarazed Shipyards, and it talks about a man named, "Goraz Zoltof," a widower in their communications research laboratory. There's an address and detailed information about the man.`
+			`	After a series of shuttles and trains, you reach his house and knock on the door.`
+			`	He's a little groggy, but manages to ask, "Yes?"`
+			`	You explain about the microtransceivers, the Quarg, and the kidnappings. You avoid mentioning that some of the people are Hai, just in case he's never heard of them.`
+			`	He seems to awaken immediately, "You found a lead on my wife's disappearance? Yes, I can help you modify your ship's communicator to detect these signals. Let's go to my laboratory at once."`
+			choice
+				`	(Head to the laboratory.)`
+			`	After another series of shuttles and trains, you reach what looks like a spaceport. From the extra equipment and unusual ship designs, you realize it is a research shipyard. Goraz's laboratory is an entire docking bay with a modified Blackbird. He takes you on board and walks you to the communications room.`
+			`	"This is a non-standard communicator, but it should be easy to reprogram yours." He takes the data card you got from the Quarg and plugs it into the communicator. After several minutes of typing, he speaks again.`
+			`	"There is nothing in this vicinity, but I was able to patch into our planet-scale receivers. They are extremely sensitive, and connected to the communication relay networks throughout human space. Those communicators aren't designed for precise tasks like this, but I can tell you where to look."`
```

Comment:
I read it as more a matter of, the comms network is capable of picking up on any unprotected general-broadcast signal. (which these are, though weak). It's not scanning for them, simply now bring told to look through what it's receiving for something it wasn't considering significant before.
Whereas in war you wouldn't be necessarily broadcasting your ship and position on unencrypted channels etc.

Reply:
It's taken me far too many attempts to write this in a way that doesn't involve pages of excessive technical detail, so it'll probably look like a bunch of unjustified assertions. I'm happy to expand on any of them if necessary, though.

The purpose of a communications network is primarily to provide efficient communications from any point in the network to any other point. The only practical way to do that is to use a hierarchical architecture with systems that deal with high-level and network-wide concerns (routing, billing, interconnection with other networks, etc.) at the top and systems that deal with low-level and very local concerns (RF signalling, channel allocation, managing interference, etc.) at the bottom. The whole system is aggressively optimised for moving comms data from one part of the network to another, with maybe a sliver of capacity left over for things like monitoring and control of the network as a whole.

As a result, someone sitting in the network operations centre only really deals with the high-level stuff and whatever predefined monitoring and control has been set up across the network as a whole. They could probably tell you that there are a surprising number of calls between Vladivostok and Nairobi today. They almost certainly couldn't tell you that all the terrestrial TV stations in Jakarta have stopped broadcasting, even though the hardware in cellphone base stations is technically capable of detecting that.

With sufficiently futuristic (or even modern) equipment, it may be possible to reconfigure base stations across the entire network to detect and report on arbitrary signals -- this is pretty much what I expect Goraz is doing -- but doing that would require diverting capacity from the core business of providing a communications service. At the very least, this would be immediately apparent to everyone looking at the system status. Most likely, there would be alarms going off and people being pulled out of bed and dropped calls all across the galaxy and hastily convened disciplinary proceedings (probably ending in termination, possibly including criminal charges) and generally a bad time all around.

(If you can do this sort of thing *without* setting off lots of alarms, all of your security and infrastructure personnel need to be fired immediately. Out of torpedo tubes. Also, you're probably so overprovisioned that you're already being outcompeted by service providers who are running more efficient networks.)

Finally: If standard communications gear can detect signals that the Hai can't see beyond 30m (even with instructions from the Quarg), it can definitely detect the waste emissions of an uncloaked Navy fleet.

Reply:
Despite all the above, I think this part of the story is fundamentally sound and with a little tweaking it could be a whole lot better. Here are some ideas:

 * We've already established that Goraz has a personal interest in the kidnappings, and he also has access to significant signal-processing capabilities. Maybe he's made contact with a bunch of other people who've had similar experiences (perhaps through Teeneep's group), and together they've been collecting broad-spectrum scans of all ships in the general vicinity of the known kidnappings. Goraz has been trying (for months, years, however long his wife has been missing) to find some pattern in the data that would identify the kidnappers. The microtransceiver information you provide lets him run a much more targeted analysis and pinpoint systems (or maybe even individual ships) in his existing dataset without having to use a galactic comms network. *This lets Goraz provide the same information he currently provides, but without the realtime surveillance issues.*
 * Maybe Goraz doesn't have a network of contacts or a bunch of data, or maybe he's just not that obsessed. Since we already know where we found the escaped Hai victims, Goraz can modify our gear and we could go back there to look for kidnappers. *This makes Goraz a little less helpful, but it links back to stuff we've already done and reinforces the continuity of the story as a whole.*
 * Not really directly related, but I found the kidnapper ships incredibly difficult to find and attack before they jumped out of the system (possibly exacerbated by flying an overstuffed Korath Dredger with a turning circle measured in parsecs). Maybe it would be interesting to have those show up as merchants (like the disguised pirate bounty missions) and have the player scan them to discover whether they're kidnappers or not. *Not sure if this is actually possible, but I think it would be a lot more fun than trying to hit a ship that frequently jumps away just as you target it.*

Reply:
Thankyou for the detailed response. Neither Nomadic nor I are comms people by any stretch. I will see what I can do with this section to align things to your ideas.

Reply:
One of those first two bullet points above still seems like the way to go to me.

Reply:
I believe I have now reworked this in line with a combo of those two points.